code/ATMOSPHERICS/components/binary_devices/circulator.dm

```diff
@@ -66,3 +66,13 @@
 		icon_state = "circ-off"
 
 	return 1
+
+/obj/machinery/atmospherics/binary/circulator/attackby(var/obj/item/W as obj, var/mob/user as mob, params)
+	if(istype(W, /obj/item/weapon/wrench))
+		var/turf/T = get_turf(src)
+		if(!istype(T, /turf/simulated))
+			return ..()
+		anchored = !anchored

```

Comment:
It generally behaves like an endpoint for most pipes as-is - but our atmos machinery code is a bit of a mess, yes.